pkg/operator/configobservation/etcdendpoints/observe_etcd_endpoints_test.go

```diff
@@ -117,12 +117,23 @@ func TestObserveStorageURLs(t *testing.T) {
 			),
 			expected: observedConfig(withStorageURL("https://10.0.0.1:2379"), withLocalhostStorageURLs()),
 		},
+		{
+			name:     "ValidIPv4NoCurrentConfig",
+			endpoint: endpoints(withAddress("10.0.0.1")),
+			expected: observedConfig(withStorageURL("https://10.0.0.1:2379"), withLocalhostStorageURLs()),
+		},
+		{
+			name:         "NoCurrentConfigNoEndPoints",
+			endpoint:     endpoints(),
+			expected:     observedConfig(withLocalhostStorageURLs()),
+			expectErrors: true,
+		},
 	}
 	for _, tt := range tests {
```

Comment:
I think that I'm going to leave these tests here and maybe I will move a subset to lib-go.